src/styles/header.scss

```diff
@@ -0,0 +1,68 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.body {
```

Comment:
The class '.body' should be renamed to a more BEM-friendly name, such as '.page' or '.main-content'.